tools/mo/requirements_tf.txt

```diff
@@ -1,5 +1,4 @@
-tensorflow~=2.5.0; python_version <= "3.6"
-tensorflow~=2.5.3; python_version > "3.6"
+tensorflow>=1.15.5,<2.6
```

Comment:
please make the similar change in `requirements.txt` file. We avoid overriding TF version in case installation the whole openvino-dev package

Reply:
Thanks a lot! done!